src/Blueprint/Blueprint/help/Az.Blueprint.md

```diff
@@ -1,28 +1,52 @@
 ---
 Module Name: Az.Blueprint
-Module Guid: {{ Update Module Guid }}
-Download Help Link: {{ Update Download Link }}
-Help Version: 1.0.0.0
-Locale: en-us
+Module Guid: ef36c942-4a71-4e19-9450-05a35843deb6
+Download Help Link: {{Please enter FwLink manually}}
+Help Version: {{Please enter version of help manually (X.X.X.X) format}}
+Locale: en-US
 ---
 
 # Az.Blueprint Module
 ## Description
-The topics in this section document the Azure PowerShell cmdlets for Azure Blueprints in the Azure Resource Manager framework. The cmdlets exist in the Microsoft.Azure.PowerShell.Cmdlets.Blueprint namespace.
+{{Manually Enter Description Here}}
```

Comment:
@filizt please make sure the template descriptions for each cmdlet is filled in in this file -- it looks like some of the descriptions were reverted